src/modules/0.0.21/aws_iam/entity/user.ts

```diff
@@ -0,0 +1,31 @@
+import { Column, CreateDateColumn, Entity, PrimaryColumn } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+
+// TODO complete schema
+@Entity()
+export class IamUser {
+  @Column({
+    nullable: true,
+  })
+  arn?: string;
+
+  // Guaranteed unique in AWS
```

Comment:
Ugh. Validation of this is next-to-impossible :(

Reply:
this is just readonly so we do not need to validate